realm/src/androidTest/java/io/realm/tests/api/RealmListTest.java

```diff
@@ -1,195 +0,0 @@
-/*
```

Comment:
I would keep these outcommented tests until they are replaced (or re-enabled and refactored)